libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/speedlimit/SpeedLimitOptions.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.androidauto.car.navigation.speedlimit
+
+import com.mapbox.navigation.base.speed.model.SpeedLimit
+import com.mapbox.navigation.base.speed.model.SpeedLimitSign
+
+/**
+ * Modify behavior of the speed limit widget.
+ */
+class SpeedLimitOptions private constructor(
```

Comment:
Will you add a `SpeedLimitOptionsTest`? The `BuilderTest` does a good job at making sure equals/hashCode are working properly